docs/cs-manual/source/record-filters.rst

```diff
@@ -0,0 +1,277 @@
+Record Filters
+##############
+
+Filters are a key element of the OpenMW CS user interface by allowing rapid and
+easy access to the searched records presented in all tables. In order to use
+this application effectively you familiarize yourself with all the concepts and
+instructions explained in this chapter of the manual. The filter system is
+somewhat unusual at first glance, but once you understand the basics it will be
+fairly intuitive and easy to use
+
+Filters are a key element to using the OpenMW CS efficiently by allowing you to
+very quickly narrow down the table entries and find what you are looking for.
+The filter system might appear unusual at first, you don't just type in a word
+and get all instances where it occurs, instead filters are first-class objects
+in the CS with their own table. This allows you to define very specific filters
+for your project and store them on disc to use in the next session. The CS
+allows you fine-grained control, you can choose whether to make a filter
+persistent between session, only for one session or use a one-off filter by
+typing it directly into the filter field.
+
+
+
+Terms used
+**********
+
+Filter
+   A Filter is generally speaking a tool able to filter the elements of a
+   table, that is select some elements while discarding others, according to
+   some criteria. These criteria are written using their own syntax.
+
+
+Criterion
+   A criterion describes some condition a record needs to satisfy in order to
+   be selected. They are written using a special syntax which is explained
+   below. We can logically combine multiple criteria in a filter for finer
+   control.
+
+Expression
+   Expressions are how we perform filtering. They look like functions in a
+   programming language: they have a name and accept a number for arguments.
+   The expression evaluates to either ``true`` or ``false`` for every record in
+   the table. The arguments are expressions themselves.
+
+Arity
+   The arity of an expression tells us how many arguments it takes. Expressions
+   taking no arguments are called *nullary*, those taking one argument are
+   known as *unary* expressions and those taking two arguments are called
+   *binary*.
+
+
+
+Interface
+*********
+
+Above each table there is a field which is used to enter a filter: either one
+predefined by the OpenMW CS developers or one  made by you. Another important
+elements is the filter table found under *View* → *Filters*. You should see the
+default filters made by the OpenMW team in the table. The table has the columns
+*Filter*, *Description* and *Modified*.
+
+ID
+   A unique name used to refer to this filter. Note that every ID has a
+   namespace prefix, we will explain these soon.
+
+Modified
+   This is the same as for records, except that it applies to filters. It tells
+   us whether the filter is base (*untouched*), *modified* or *removed*.
+   Filters are specific to a project instead of a content file, they have no
```

Comment:
Actually, filter can not have base state and neither can they have modified or moved. We may allow at some point the inclusion of filters in the content files. But for now the only possible modification state is added.